docs/docs/concepts/components/overview.mdx

```diff
@@ -0,0 +1,62 @@
+---
+id: overview
+sidebar_label: Overview
+title: Model Configuration
+description: Learn about model configuration for Rasa.
+abstract: The configuration file defines the components and policies that your model will use to make predictions based on user input.
+---
+
+The recipe key allows for different types of config and model architecture.
+Currently, "default.v1" and the experimental "graph.v1" recipes are supported.
```

Comment:
we probably need to rework this intro, right? or at least give more context (with an example config file, etc...)

Reply:
yes, I have a todo because first we actually have to figure out what to do with "autoconfig" in a calm world